src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -932,3 +978,87 @@ button.link {
     max-width: 200px;
     width: 100%;
 }
+
+/* Right Panel */
+
+.RoomInfo {
```

Comment:
So, as said, the CSS naming conventions are currently wildly inconsistent. But for CSS classes that are not used in multiple views (which I think applies to all the below?), I do try to use the convention of `ViewName` for the root element, and `ViewName_something` for sub elements.

Reply:
As commented in RoomInfoViewModel, I'd suggest the name `RoomDetailsView` here.

I imagine once you add the member list to the right panel. RoomDetailsView would be one possible child of RightPanelView or something, with `MemberListView` being the other possibility.